2 - RTB/Documentazione esterna/Piano di Qualifica/Piano di Qualifica.typ

```diff
@@ -251,9 +268,211 @@ Il presente documento include una serie di termini tecnici specifici del progett
+== Processi primari
+=== Fornitura
+==== Rapporto tra PPV, PAC e PEV
+#figure(
+  cetz.canvas({
+    import cetz.plot 
+    let EV_points(offset: 0) = ((4,2040), (5,2655), (6,3111.85), (7,3528.52), (8,3948.14), (9, 4573.14), (10, 4848.14), (11, 5084.39), (12, 5178.90), (13, 5224.62)).map(((x,y)) => {(x,y + offset * 1.5)})
+    
+    let AC_point(offset: 1) = ((4,2075), (5,2620), (6,3140), (7,3515), (8,4090), (9, 4520), (10, 4875), (11, 5105), (12, 5285), (13, 5395)).map(((x,y)) => {(x,y + offset * 1.5)})
+
+    let PV_point(offset: 1) = ((4,2040), (5,2655), (6,3190), (7,3690), (8,4200), (9, 4825), (10, 5155), (11, 5470), (12, 5625), (13, 5705)).map(((x,y)) => {(x,y + offset * 1.5)})
+
+    plot.plot(size: (12, 6), {
+    plot.add(PV_point(offset: 1), line: "spline", label: "PPV")
+    plot.add(AC_point(offset: 1), line: "spline", label: "PAC", style: (stroke: (paint: red)))
+    plot.add(EV_points(offset: 0), line: "spline", label: "PEV", style: (stroke: (paint: green)))
+    plot.add-hline(13055, label: "BAC" , style: (stroke: (paint: blue, dash: "dotted")))
+    plot.add-vline(13, label: "RTB", style: (stroke: (paint: black, dash: "dotted")))
+    plot.add-vline(20, label: "PB" , style: (stroke: (paint: red, dash: "dotted")))
+    },
+    y-max: 14000,
+    x-max: 21,
+    x-min: 4,
+    x-tick-step: 1,
+    y-tick-step: 1500,
+    x-label: "Sprint",
+    y-label: "Valore in €",
+    )
+  }),
+  caption: "Rapporto tra PPV, PAC e PEV"
+)
+*RTB*: In questo primo periodo, il gruppo è consapevole che il valore pianficato *PPV* risulta superiore a quanto prodotto nell'effettivo indicato dal *PEV*. Nonostante ciò il gruppo è sempre riuscito a mantenere il valore del *PEV* non solo in crescita, ma anche superiore almeno all'80% del *PPV*.
+
+#pagebreak()
+
+==== Cost Performance Index CPI
+#figure(
+  cetz.canvas({
+    import cetz.plot 
+    let CPI_points(offset: 0) = ((4,0.98), (5,1.01), (6,0.99), (7,1.00), (8,0.97), (9, 1.01), (10, 0.99), (11, 1.00), (12, 0.98), (13, 0.97)).map(((x,y)) => {(x,y + offset * 1.5)})
+
+    plot.plot(size: (12, 6), {
+    plot.add(CPI_points(offset: 0), line: "linear", label: "CPI", mark: "triangle")
+    plot.add-hline(1.1, label: "Limite superiore" , style: (stroke: (paint: green, dash: "dotted")))
+    plot.add-hline(0.8, label: "Limite inferiore" , style: (stroke: (paint: red, dash: "dotted")))
+    plot.add-vline(13, label: "RTB", style: (stroke: (paint: black, dash: "dotted")))
+    plot.add-vline(20, label: "PB" , style: (stroke: (paint: red, dash: "dotted")))
+    },
+    y-max: 1.2,
+    y-min: 0.6,
+    x-max: 21,
+    x-min: 4,
+    x-tick-step: 1,
+    y-tick-step: 0.1,
+    x-label: "Sprint",
+    y-label: "CPI",
+    )
+  }),
+  caption: "Andamento CPI"
+)
+*RTB*: L'indice *CPI* risulta sempre in un range di valore accettabile. Seppur non lineare, non si evidenziano grandi variazioni, permettendo di evidenziare un corretto avanzamento in termini di costi e lavoro prodotto.
```

Comment:
C'è una ripetizione di "evidenziare". Potresti scrivere "Seppur l'andamento non sia lineare, non si rilevano grandi variazioni, il che evidenzia un corretto avanzamento in termini di costi e lavoro prodotto"